mbtb_app/web_portal/mbtb_portal/config/routes.js

```diff
@@ -292,4 +292,33 @@ module.exports.routes = {
     }
   },
```

Comment:
5 -> Have you removed previous routes from here for deleted controllers? if no, please do it.